utils/create_fhir_store.sh

```diff
@@ -22,7 +22,34 @@ if [[ $# -ne 4 ]]; then
   exit 1
 fi
 
-# TODO create GCP Healthcare data-set too instead of assuming it exists.
+# Function that creates dataset and checks if the dataset already exist
+function create_dataset
```

Comment:
nit: I think the [style guide](https://google.github.io/styleguide/shellguide.html) recommendation is something like:
```
function create_dataset() {
``` 
or
```
create_dataset() {
```